src/components/Sum.jsx

```diff
@@ -1 +1,5 @@
-// export function Sum() {}
+import React from 'react';
+
+const Sum = ({ a = 0, b = 0 }) => (<p>{`Sum of ${a} and ${b} is ${a + b}`}</p>);
```

Comment:
Please note: use named export (not default).
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (<p>{`Sum of ${a} and ${b} is ${a + b}`}</p>);
```